src/components/sbb-timetable-row/sbb-timetable-row.tsx

```diff
@@ -164,7 +164,12 @@ export class SbbTimetableRow {
 
     return (
       <Host role="rowgroup">
-        <sbb-card aria-expanded={this.accessibilityExpanded} size="l" active={this.active} id={id}>
+        <sbb-card
```

Comment:
Is the timetable row always used with expanded prop? (True or false). If yes, it‘s ok, but otherwise we better change the type to a optional string literal. With that you could opt out of setting expanded

Reply:
yes.. currently its only used like that.

Reply:
Are you sure also about other projects? maybe it's worth to change it to be safe?

Reply:
Will refactor it anyways in card refactoring.